extensions/sql-migration/src/api/azure.ts

```diff
@@ -259,6 +280,19 @@ export async function getAvailableSqlVMs(account: azdata.Account, subscription:
 	return response.response.data.value;
 }
 
+export async function getVMInstanceView(sqlVm: SqlVMServer, account: azdata.Account, subscription: Subscription): Promise<VirtualMachineInstanceView> {
+	const api = await getAzureCoreAPI();
+	const path = encodeURI(`/subscriptions/${subscription.id}/resourceGroups/${getResourceGroupFromId(sqlVm.id)}/providers/Microsoft.Compute/virtualMachines/${sqlVm.name}/instanceView?api-version=2022-08-01`);
+	const host = api.getProviderMetadataForAccount(account).settings.armResource?.endpoint;
```

Comment:
As we're using `armResource?`, would host be `null` here?

Reply:
Same as my other comment - agreed the error handling could probably be improved here, but I reused the same implementation as all the other custom REST API requests we make here, and haven't seen any issues in the past

Reply:
We could probably take that as a follow up item to improve the error handling on all the API calls, but doesn't seem super high pri for now